src/scripts/Display/cards.js

```diff
@@ -0,0 +1,55 @@
+/* eslint-disable no-restricted-syntax */
```

Comment:
- [ ] Please remove this eslint disable line its best to fix the issue in the code.